### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jasonbasuil**
Created a new issues template (borrowed from RX Workflow)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/-ui-design-system--issue.md
Comment (kyleshike) [inline .github/ISSUE_TEMPLATE/-ui-design-system--issue.md:1]: What do you think about adding some checkboxes for: 

- specs
- storybook documentation
- a11y

Over in RS, it's been a nice way to nudge devs toward ensuring that they're writing all the supporting code for their given feature work.

**Review by kyleshike — APPROVED** [review]
Looks great, just a few questions and one suggestion.  But I also think this is something we can easily iterate on if need be

File: .github/ISSUE_TEMPLATE/-ui-design-system--issue.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## User Stories
+
+- As a [persona], I want [feature/task] so that [benefit/reason].
+
+## Acceptance Criteria
+
+- [Criteria 1]
+- [Criteria 2]
+- [Criteria 3]
```
[the remark this answers is not in the document]
Comment (jasonbasuil) [reply .github/ISSUE_TEMPLATE/-ui-design-system--issue.md:31]: Perhaps: 

- The Accordion is able to collapse and open
- The Accordion chevron is able to rotate -180 degrees when opening and closing

Something along those lines?

[the remark this answers is not in the document]
Comment (kyleshike) [reply .github/ISSUE_TEMPLATE/-ui-design-system--issue.md:31]: Great makes sense!

**Pull request merged**